messaging/src/main/java/org/axonframework/eventhandling/GlobalSequenceTrackingToken.java

```diff
@@ -31,6 +31,8 @@
 public class GlobalSequenceTrackingToken implements TrackingToken, Comparable<GlobalSequenceTrackingToken>,
                                                     Serializable {
 
+    private static final long serialVersionUID = -3658606104934080049L;
```

Comment:
The same concern as for `GapAwareTrackingToken`.

Reply:
See previous comment on the `GapAwareTrackingToken`.